packages/headless-components/cms/src/services/cms-item-service.ts

```diff
@@ -0,0 +1,171 @@
+import { defineService, implementService } from '@wix/services-definitions';
+import {
+  SignalsServiceDefinition,
+  type Signal,
+} from '@wix/services-definitions/core-services/signals';
+import { items } from '@wix/data';
+import { type WixDataItem } from './cms-collection-service.js';
+
+/**
+ * Service definition for the CMS Item service.
+ */
+export const CmsItemServiceDefinition = defineService<{
+  /** Reactive signal containing the individual item */
+  itemSignal: Signal<WixDataItem>;
+  /** Reactive signal indicating if the item is currently being loaded */
+  loadingSignal: Signal<boolean>;
+  /** Reactive signal containing any error message, or null if no error */
+  errorSignal: Signal<string | null>;
+}>('CmsItem');
+
+/**
+ * Shared function to load a single item from Wix Data
+ */
+const loadItem = async (
+  collectionId: string,
+  itemId: string,
+): Promise<WixDataItem> => {
+  if (!collectionId) {
+    throw new Error('No collection ID provided');
+  }
+
+  if (!itemId) {
+    throw new Error('No item ID provided');
+  }
+
+  const result = await items.get(collectionId, itemId);
+
+  if (!result) {
+    throw new Error('Item not found');
+  }
+
+  return result;
+};
+
+/**
+ * Configuration interface required to initialize the CmsItemService.
+ */
+export interface CmsItemServiceConfig {
+  /** The collection ID to load the item from */
+  collectionId: string;
+  /** The item ID to load */
+  itemId: string;
+  /** Optional pre-loaded item to initialize the service with */
+  item?: WixDataItem;
+}
+
+/**
+ * Implementation of the CMS Item service that manages reactive item data.
+ */
+export const CmsItemServiceImplementation =
+  implementService.withConfig<CmsItemServiceConfig>()(
+    CmsItemServiceDefinition,
+    ({ getService, config }) => {
+      const signalsService = getService(SignalsServiceDefinition);
+
+      // Initialize with pre-loaded item if provided, otherwise with empty object
+      const itemSignal: Signal<WixDataItem> = signalsService.signal<WixDataItem>(
+        config.item || ({ _id: '' } as WixDataItem),
+      );
+      const loadingSignal: Signal<boolean> = signalsService.signal<boolean>(false);
```

Comment:
You don't need the double type assertion
```suggestion
      const loadingSignal = signalsService.signal<boolean>(false);
```